SmartTutor/Startup.cs

```diff
@@ -47,6 +53,42 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<ITraineeRepository, TraineeDatabaseRepository>();
             services.AddScoped<IFeedbackRepository, FeedbackDatabaseRepository>();
             services.AddScoped<IRecommender, KnowledgeBasedRecommender>();
+
+            services.AddAuthentication(options =>
```

Comment:
Can we extract this setup into a private function (e.g., ConfigureAuthentication)?